client/src/app/services/canvas/canvas.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+export class CanvasService {
+
+  private headers: HttpHeaders = null;
+  private endpointBase: string = null;
+
+  constructor(private httpClient: HttpClient) { 
+    this.endpointBase = '/testing'; 
```

Comment:
Also oops. Will fix this before I merge. Should be `this.endpointBase = `'/api/v1/signature'`

Reply:
Note to self: lose the v1